test/index.spec.js

```diff
@@ -5,16 +5,23 @@ let parser = require('../src/parser');
 
 describe('loader', () => {
   let cacheableCalled = false;
-  let loaderContext = { cacheable: () => { cacheableCalled = true } };
+  let loaderContext = {
+    cacheable: () => { cacheableCalled = true },
+    options: {
+      requirejsPlugins: {
+        plugins: ['plugin']
+      }
+    }
+  };
 
   it('parses files with plugins', () => {
```

Comment:
[re: optional vs required options] What happens if you have no plugins in the config, or no config defined at all in the loaderContext?